variables.tf

```diff
@@ -85,3 +85,36 @@ variable "graylog_port" {
 variable "graylog_cidr" {
   description = "Cidr Block for Graylog"
 }
+
+variable "task_cpu" {
+  default = "256"
+  description = "The number of cpu units to reserve for the task, must be higher than the all the CPU in the containers in the task"
+}
+
+variable "task_memory" {
+  default = "512"
+  description = "The amount of memory (in MiB) to allow the task to use. Must be higher than all the containers in the task"
+}
+
+variable "container_memory" {
+  type        = number
+  default     = "256"
```

Comment:
Doesn't this mean you've got an extra 128mb of memory nothing can use?

In that `task_memory` is 512, then Datadog is set to a 128mb hard limit and the app is set to 256mb by default, leaving 128mb over

Reply:
Good point, will up the amount available to our application container by default.